### ISSUE TITLE
Radarr fails to run after update

### ISSUE THREAD

**Issue by narolski** [MichaIng/DietPi#1922]
#### Required Information:
- DietPi version	| 6.11
- Distro version	| stretch
- Kernel version	| Linux DietPi 4.14.52-v7+ #1123 SMP Wed Jun 27 17:35:49 BST 2018 armv7l GNU/Linux
- SBC device		|  RPi 3 Model B (armv7l)
- Power supply used	| Aukey (5V, 3A)
- SDcard used		| Samsung Evo

#### Additional Information (if applicable):
- Software title	| Radarr
- Was the software title installed freshly or updated/migrated? | No
- Can this issue be replicated on a fresh installation of DietPi? | 
- `dietpi-bugreport` ID | 487cf4e0-fd7e-4269-9697-1c7e5085f016

#### Steps to reproduce:
Radarr service fails to load.

#### Expected behaviour:
Radarr should work fine and allow access to the GUI.

#### Actual behaviour:
```
● radarr.service - Radarr Daemon
   Loaded: loaded (/etc/systemd/system/radarr.service; disabled; vendor preset: enabled)
   Active: failed (Result: exit-code) since Sat 2018-07-14 22:17:42 CEST; 6min ago
  Process: 1298 ExecStart=/usr/bin/mono --optimize=all --server /opt/Radarr/Radarr.exe -nobrowser (code=exited, status=1/FAILURE)
 Main PID: 1298 (code=exited, status=1/FAILURE)

Jul 14 22:17:41 DietPi mono[1298]:   at NzbDrone.Core.Configuration.ConfigFileProvider.GetValue (System.String key, System.Object defaultValue, System.Boolean persist) [0x00022] in <d622740c3f6a4ed084b707bca6d9b562>:0
Jul 14 22:17:41 DietPi mono[1298]:   at NzbDrone.Core.Configuration.ConfigFileProvider.get_LogLevel () [0x00000] in <d622740c3f6a4ed084b707bca6d9b562>:0
Jul 14 22:17:41 DietPi mono[1298]:   at NzbDrone.Core.Instrumentation.ReconfigureLogging.Reconfigure () [0x00000] in <d622740c3f6a4ed084b707bca6d9b562>:0
Jul 14 22:17:41 DietPi mono[1298]:   at Radarr.Host.Bootstrap.Start (Radarr.Host.ApplicationModes applicationModes, NzbDrone.Common.EnvironmentInfo.StartupContext startupContext) [0x0000a] in <995dab1107b24482ba8bf8963e3706e7>:0
Jul 14 22:17:41 DietPi mono[1298]:   at Radarr.Host.Bootstrap.Start (NzbDrone.Common.EnvironmentInfo.StartupContext startupContext, Radarr.Host.IUserAlert userAlert, System.Action`1[T] startCallback) [0x00075] in <995dab1107b24482ba8bf8963e3706e7>:0
Jul 14 22:17:41 DietPi mono[1298]:   at NzbDrone.Console.ConsoleApp.Main (System.String[] args) [0x0002f] in <b6e6e83031db400582b8bc076d3bc359>:0
Jul 14 22:17:41 DietPi mono[1298]: Press enter to exit...
Jul 14 22:17:42 DietPi systemd[1]: radarr.service: Main process exited, code=exited, status=1/FAILURE
Jul 14 22:17:42 DietPi systemd[1]: radarr.service: Unit entered failed state.
Jul 14 22:17:42 DietPi systemd[1]: radarr.service: Failed with result 'exit-code'.
```

#### Extra details:
Tried reinstalling: NzbDrone, Sonarr, Radarr and Deluge. Didn't work.

**Comment by MichaIng** [comment]
@maxster256 
Might be related to: https://github.com/Fourdee/DietPi/issues/1896
- Please try to remove `--optimize=all --server` from
  - `/etc/systemd/system/radarr.service`
  - `/etc/systemd/system/sonarr.service`
  - `/etc/systemd/system/jackett.service`, if you installed it
- and then restart these services.

Also the full output of mono trying to start the service(s) would be helpful:
`journalctl -t mono` (or if this is empty `journalctl -u mono`
- Then all related messages after last restart/start attempt.

**Comment by Fourdee** [comment]
@MichaIng 

Yep, this is same issue. I'll mark this as closed in favor of : https://github.com/Fourdee/DietPi/issues/1896

@maxster256 

A fix is coming this week in v6.12. In the mean time, please see https://github.com/Fourdee/DietPi/issues/1922#issuecomment-405088725
